def/flow.js

```diff
@@ -164,13 +168,23 @@ def("Identifier")
 def("TypeParameterDeclaration")
   .bases("Node")
   .build("params")
-  .field("params", [def("Identifier")]);
+  .field("params", [or(def("Identifier"), def("TypeParameter"))]);
 
 def("TypeParameterInstantiation")
   .bases("Node")
   .build("params")
   .field("params", [def("Type")]);
 
+def("TypeParameter")
+  .bases("Identifier")
```

Comment:
Fun fact: I used `Type` first but switched to `Identifier` because I saw [this](https://github.com/facebook/flow/blob/master/src/parser/test/custom_ast_types.js#L6-L10) :)